featuregates/client/pkg/featuregateclient/client.go

```diff
@@ -8,33 +8,34 @@ import (
 	"fmt"
 
 	corev1 "k8s.io/api/core/v1"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
 	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/client-go/rest"
 	"k8s.io/client-go/tools/clientcmd"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 
-	configv1alpha1 "github.com/vmware-tanzu/tanzu-framework/apis/config/v1alpha1"
+	core "github.com/vmware-tanzu/tanzu-framework/apis/core/v1alpha2"
```

Comment:
I don't think it helps readability to add version, but I guess it's consistent with how we name these for metav1, corev1, etc. But I thought just doing `core` without version would distinguish it from tanzu core vs k8s core. Anyway, if you think it's better to add version, let me know. I'll go ahead and replace all `core` with `corev1alpha`. Seems like a lot of letters that hinders reading and doesn't really help with reading, but maybe you have a different perspective.

Reply:
After speaking with @yharish991, I decided it's better to have versions because what if the plugin had to support more than one version of the same group? In that case, we need to have the version in alias name.